features/api/backend_usages/index.feature

```diff
@@ -46,8 +46,8 @@ Feature: Product > Integration > Backends
 
     Scenario: Editing a backend config
       When they go to the backends of product "My API"
-      And follow "Edit config with Backend 1"
-      Then the current page is the edit usage config page between "My API" and "Backend 1"
+      And follow "Edit config with Backend 2"
+      Then the current page is the edit usage config page between "My API" and "Backend 2"
```

Comment:
How does selecting backend 2 help?

Reply:
That was a test. Since I saw the test stable eventually I thought not to revert it to `1` as I don't see a difference for the test which one it uses. If you don't like it, I can try to revert.